source/includes/api/v3/_campaigns.md

```diff
@@ -629,3 +629,303 @@ skip | true | Data offset index. Useful for pagination.
+
+Allowed values:
+
+* `'appointment'`
+* `'not-contacted-high-priority'`
+* `'not-contacted-do-not-knock'`
+* `'not-contacted-normal'`
+* `'contacted-available-successful'`
+* `'contacted-available-not-successful'`
+* `'contacted-unavailable-not-home'`
+* `'contacted-unavailable-other'`
+* `'come-back'`
+* `'unknown'`
+
+### data/statusId
+
+The statusId property must point to the id property of a status object.
+This status' `contactState` property must have the same value of the contact's `state` property, otherwise it's id will not be a valid statusId for the contact object.
+
+
+### data/availability
+
+Attribute | Required? | Description
+--------- | --------- | -----------
+available | false | Flag indicating whether the contact at the household was available or not
+unavailableReason | false | Populated if the contact was unavailable
+
+### data/reservation
+
+Attribute | Required? | Description
+--------- | --------- | -----------
+reserved | true | Boolean indicating if this contact is reserved for some route.
+walklistId | false | Route identifier this contact is reserved to.
+
+### meta
+
+[See documentation](#metadata-object).
+
+## Get contact household campaign record
+
+
+> Example Request
+
+```http
+GET /v1/campaigns/c92162c6-f604-4caf-a964-d668d8bf67c8/contacts/cc20ab46-6233-491d-82ac-94c7ba6a8997/households/e50be735-cfe3-49db-8fca-e03f630d4a7e/status HTTP/1.1
+Host: api.beta.polisapp.com
+Authorization: Bearer {access_token}
+``` 
+
+> Example Response
+
+```json
+{
+    "id": "00001820-1329-490c-b07c-2071c29329de",
+    "data": {
+        "contacted": false,
+        "campaignId": "c92162c6-f604-4caf-a964-d668d8bf67c8",
+        "successful": false,
+        "reservation": {
+            "reserved": false
+        },
+        "availability": {
+            "available": false,
+            "unavailableReason": null
+        },
+        "contactHouseholdId": "b66e4452-5fcd-4b79-bb1c-068ed7353192",
+        "state": "not-contacted-normal",
+        "statusId": "54aa7cef-877a-44ec-b2c5-442b43f5d758"
+    },
+    "meta": {
+        "etag": "295-1n/IqPXKKnZp0KmvaPoq7YWAIUI",
+        "created": "2019-09-05T20:54:33.580Z",
+        "modified": "2019-09-05T20:54:33.580Z",
+        "resource": "contact-household-campaigns",
+        "createdBy": "f7db3e84-e0f0-4b8b-94b2-2e5675ddddaa",
+        "isDeleted": false,
+        "modifiedBy": "f7db3e84-e0f0-4b8b-94b2-2e5675ddddaa"
+    },
+    "customerId": "bd4b51e6-804c-497d-8f60-cf7b12c56e06",
+    "securityGroupId": "2e61dbbc-02f2-4084-b36e-c3d1d39ed495"
+}
+```
+### HTTP Request
+
+`GET https://api.beta.polisapp.com/v1/campaigns/{campaignId}/contacts/{contactId}/households/{householdId}`
+
+### Query Parameters
+
+Parameter | Required | Description
+--------- | -------- | -----------
+campaignId | true | The identifier of a campaign.
+contactId | true | The identifier of a contact.
+householdId | true | The identifier of a household.
+
+## Update a contact household campaign record
```

Comment:
you can only update the status here, no? or everything?

Reply:
can we be more specific?

Reply:
well apparently you can update anything under data , see the example , i actually made the request and it seems to work

Reply:
question is should we add restrictions ? here saying you can only update statusId ?

Reply:
yes let's change this title and description to suggest that you should only be using it to update statusId. what about state and contacted, though? i forget whether these are auto calculated?